### ISSUE TITLE
Screengrab error on opening software

### ISSUE THREAD

**Issue by Emanuele-rgb** [macaroni-os/macaroni-funtoo#21]
Error when launching:
Cannot mix incompatible Qt library (5.15.10) with this library (5.15.11)
Aborted

**Comment by geaaru** [comment]
If the subsets `devel` and `portage` are enable a temporary workaround is this:
```bash
$> whip h qt5.qconfig_gen
$> emerge x11-misc/screengrab
```

**Comment by geaaru** [comment]
Sorry, also this library binding:
```
$> emerge dev-libs/libqtxdg
```

**Comment by geaaru** [comment]
Bumped version `app-misc/qtxdg-tools-3.11.0+1` that will be available in release 23.11.1. Thanks for the report.
In `macaroni-phoenix-testing` in few days.
